scripts/cookbook.js

```diff
@@ -0,0 +1,11 @@
+// Description
+//   Returns a link to the UQCS student-compiled cookbook (full of tasty treats).
+// Commands
+//   `!cookbook` - returns the URL of the UQCS student-compiled cookbook (pdf).
+//
+
+module.exports = function (robot) {
+  robot.respond(/!?cookbook/i, function (res) {
+    res.send("https://jenkins.uqcs.org.au/job/cookbook/lastSuccessfulBuild/artifact/main.pdf");
```

Comment:
Can you make it link to https://github.com/UQComputingSociety/cookbook instead? there's a link to the compiled PDF there but it has all the extra info too, so it's probs a better option.